src/riak_ts.proto

```diff
@@ -111,6 +112,7 @@ message TsCell {
   optional sint64 timestamp_value = 3;
   optional bool boolean_value = 4;
   optional double double_value = 5;
+  optional bytes blob_value = 6;
```

Comment:
This will definitely need an upgrade/downgrade test, with the added flag with queries, we might be in a position where the clients can't handle upgraded messages from TS because of this change and TS can't handle upgraded client messages because of the new flag on the query message.